config/locales/assessor_interface.en.yml

```
@@ -172,6 +172,8 @@ en:
               blank: Enter the subject
             selected_failure_reasons:
               blank: Select the reasons for your recommendation
+            passed:
+              inclusion: Choose whether the applicant completed this section to your satisfaction
```

```suggestion
              inclusion: Select whether the applicant completed this section to your satisfaction
```

I think the convention is to use the word select.

---

Changed 👍